Resources/Textures/Objects/Specific/Xenoarchaeology/item_artifacts.rsi/meta.json

```diff
@@ -0,0 +1,162 @@
+{
+  "version": 1,
+  "license": "CC-BY-NC-SA 3.0",
+  "copyright": "goonstation at 081ed42cf5386cc6649ac290cb48de4ba2e5cf40",
```

Comment:
Goon's picky so make sure you find the exact source commit, not just any commit. Also ask.

Reply:
![image](https://user-images.githubusercontent.com/19853115/202514698-4075159c-23d7-42da-b040-31365b836d61.png)